service/api_logic/news_logic.py

```diff
@@ -37,6 +37,18 @@ def get_popular_news(count: int, session):
     news = fetch_news(session, order_by=News.interest_rate.desc(), limit=count)
     return json_news(news)
 
+@handle_exceptions
+def get_news_by_id(blob_id: str, session):
+    news = fetch_news(session, filters=[News.blob_id == blob_id], limit=1)
+    if news:
+        return json_news(news)
+    else:
+        return Response(
```

Comment:
Business logic must not return any response.

It either returns the result, or yields an exception